Cargo.toml

```
@@ -35,14 +34,14 @@ sea-orm = { version = "1.1.7", features = [
     "runtime-tokio-native-tls",
     "macros",
 ] }
-aws-config = "1.6"
-aws-sdk-secretsmanager = "1.68"
-
+aws-config = "0.55.3"
+aws-sdk-secretsmanager = "0.26.0"
+aws-secretsmanager-cache = "0.5.0"
```

You are using outdated versions of these crates. Is there a reason for this? If not, it is advisable to go to the latest available versions.
Additionally, make a habit to specify versions that are subject to breaking changes ( as major sometimes minor versions).
Use aws-secretsmanager-caching instead of aws-secretsmanager-cache which is not longer maintained.
```suggestion
aws-config = "1.6"
aws-sdk-secretsmanager = "1.71"
aws-secretsmanager-caching = "1.2"
```